docs/book/src/clusterctl/provider-contract.md

```diff
@@ -114,7 +114,12 @@ recommended to prefix the variable name with the provider name e.g. `${ AWS_CRED
 
 Additionally, each provider should create user facing documentation with the list of required variables and with all the additional
 notes that are required to assist the user in defining the value for each variable.
- 
+
+#### Labels
+The components YAML components should be labeled with
+`cluster.x-k8s.io/provider` and the appropriate value. This will enable an
```

Comment:
We need to have a table with label name -> description or something similar, even a list would be ok I think

Reply:
Do you mean like the table in `adopt.md`? 
I intentionally decided to not add that table here since this section describes the contract for provider that may be implemented. We could however provide a suggestion for what the value may be as per your guideline of `<type>-<name>`.
That is provide an example like `cluster.x-k8s.io/provider=infrastructure-myIaas`

Reply:
appropriate name --> and the name of the provider